src/pages/lessons/[slug].tsx

```diff
@@ -579,7 +581,16 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
               )}
               {playerState.matches('addingNote') && (
                 <OverlayWrapper>
-                  <AddNoteOverlay onClose={() => send('VIEW')} />
+                  <AddNoteOverlay
+                    resourceId={lesson.slug}
+                    onClose={(notes: any) => {
+                      setNewNotes(notes)
```

Comment:
the replaces `newNotes` with a fresh array containing the notes that just got created

Reply:
Should this be appending notes to an array of _new notes_? Or is it supposed to be the latest note?

Reply:
it's always an array and it's also always only one note as presently implemented